integration-tests/tests/resource-observer/resource-obs.spec.js

```diff
@@ -0,0 +1,50 @@
+/*
+Copyright 2020 Splunk Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+module.exports = {
+  'should report resource loads happening after page load': async function(browser) {
+    browser.globals.clearReceivedSpans();
+    await browser.url(browser.globals.getUrl('/resource-observer/resources.ejs'));
+
+    const imageSpan = await browser.globals.findSpan(
```

Comment:
You should send a guard span at the point when the page is loaded and wait for it here. You can grep the project by "guard", and just copy paste the implementation.

Reply:
I actually do use it for the ignore test already but good idea, I'll use it here also.

Reply:
I don't see any guard spans in the ignore url test. What I mean is https://github.com/signalfx/splunk-browser-otel-js/blob/main/integration-tests/tests/fetch/fetch.spec.js#L46

Reply:
Right, my bad.